docs/developer/contributing/development-functional-tests.asciidoc

```diff
@@ -62,7 +62,7 @@ export TEST_ES_PASS=<password>
 node scripts/functional_test_runner
 ----------
 
-** If you are running x-pack functional tests, start server and runner from {blob}xpack[x-pack] folder:
+** If you are running x-pack functional tests, start server and runner from the link:{kib-repo}/tree/{branch}/x-pack[x-pack] folder:
```

Comment:
Same as above, the resulting link is currently https://github.com/elastic/kibana//tree/7.10/x-pack

```suggestion
** If you are running x-pack functional tests, start server and runner from the link:{kib-repo}tree/{branch}/x-pack[x-pack] folder:
```